### PR TITLE
Fix for data race

### PR DESCRIPTION
**Pull request by clstrfsck**

### CODE REVIEW

File: radixdlt/src/main/java/org/radix/api/services/AtomsService.java [hunk 1]
```diff
@@ -65,19 +68,21 @@
 import com.radixdlt.identifiers.AID;
 
 public class AtomsService {
-	private static int  NUMBER_OF_THREADS = 8;
+	private static final int  NUMBER_OF_THREADS = 8;
```
Comment (LukasGasior1) [inline radixdlt/src/main/java/org/radix/api/services/AtomsService.java]: super minor: are there two spaces?

**Review by siy — APPROVED** [review]
LGTM

**Review by LukasGasior1 — APPROVED** [review]
LGTM

File: radixdlt/src/main/java/org/radix/api/services/AtomsService.java (hunk 1 above)
Comment (clstrfsck) [reply radixdlt/src/main/java/org/radix/api/services/AtomsService.java]: Yeap.  Fixed.

**Review by talekhinezh — APPROVED** [review]

**Pull request merged**